app-misc/recoll/recoll-1.31.0.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit linux-info optfeature python-single-r1 qmake-utils
+
+DESCRIPTION="Personal full text search package"
+HOMEPAGE="https://www.lesbonscomptes.com/recoll/"
+SRC_URI="https://www.lesbonscomptes.com/recoll/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="camelcase chm doc +inotify qt5 session +spell webengine"
```

Comment:
I don't think we need `doc` here. Can you check how much worth diskspace the html docs are when installed? If only a few KBs, just install them unconditionally.